iotile_ext_cloud/iotile/__init__.py

```diff
@@ -1,2 +1,6 @@
 from pkgutil import extend_path
+from cloud.utilities import device_slug_to_id, device_id_to_slug
+from cloud.cloud import IOTileCloud
+
 __path__ = extend_path(__path__, __name__)
+__all__ = ['device_slug_to_id', 'device_id_to_slug', 'IOTileCloud']
```

Comment:
This needs to go in `iotile.cloud`, one level deeper.  The coretools packages use a namespace package approach so that everything is importable under a standard `iotile.` namespace even though `iotile.core` and `iotile.cloud` are provided by different pip installable distributions.  

As a consequence though, you can't put anything in the root `iotile` package since it might get overwritten by other packages that also claim the `iotile` namespace package.  See:
https://packaging.python.org/guides/packaging-namespace-packages/#pkg-resources-style-namespace-packages

Reply:
Duly noted, edited !